clusterloader2/README.md

```diff
@@ -1,6 +1,19 @@
 # ClusterLoader
 
-#### Vendor
+### Running ClusterLoader
+
+To run ClusterLoader type:
+```
+go run cmd/clusterloader.go --clientconfig=kubeConfig.yaml --testconfig=config.json
```

Comment:
Please rename this flag to `--kubeconfig` (to match the convention everywhere else).

Reply:
ok.